config/config.go

```diff
@@ -201,6 +204,10 @@ func UnregisterProviders() {
 // Load creates a Provider for use in a service
 func Load() Provider {
 	var static []Provider
+
+	if len(_configFiles) == 0 {
```

Comment:
Do we need to? You can simply not have config files in the /config dir. The default is here just to tell the yaml provider where to look for configurations.